src/Modules/Station/Components/PlaylistTabs/Playlist/Playlist.tsx

```diff
@@ -1,18 +1,28 @@
+import { Song } from 'Models/Song';
 import * as React from 'react';
 import { Component } from 'react';
 import FlipMoveList from 'react-flip-move';
 import { Card, CardBody } from 'reactstrap';
+import { UserServices } from 'Services/Http/UserServices';
+import { IFavouriteItem } from '../Favourite/FavouriteItem';
 import './Playlist.scss';
 import { PlaylistItem } from './PlaylistItem';
 
 interface IPlaylistProps {
   playlist: any[];
 }
 
-export class Playlist extends Component<IPlaylistProps, IPlaylistProps> {
+interface IFavoriteListProps {
+  favoriteList: IFavouriteItem[];
+}
+
+type Iprops = IPlaylistProps & IFavoriteListProps;
+
+export class Playlist extends Component<Iprops, IPlaylistProps> {
```

Comment:
@hau-tran 
the second one is NOT `IPlayistProps`

It is the internal state of component